src/database/migrations/20220309061357-create-user.js

```diff
@@ -1,35 +1,39 @@
+'use strict';
```

Comment:
Please remove this. It's not important as we are using ES6 and causes linting error

Reply:
> Please remove this. It's not important as we are using ES6 and causes linting error

Fixed This Issue.